README.md

```diff
@@ -91,3 +91,50 @@ search_engine_solr:
             entry_endpoints:
                 - endpoint0
 ```
+
+
+## Redis Cache
+If a Platform.sh relationship named `rediscache` is defined, it will be taken as a the storage engine for a cache pool. 
```

Comment:
```suggestion
## Redis Cache

If a Platform.sh relationship named `rediscache` is defined, it will be taken as a the storage engine for a cache pool. 
```